corehq/apps/app_manager/tests/test_suite_remote_request.py

```diff
@@ -562,3 +562,62 @@ def test_prompt_default_value(self, *args):
         </partial>
         """
         self.assertXmlPartialEqual(expected, suite, "./remote-request[1]/session/query/prompt[@key='name']")
+
+    def test_cowin_appointment_search(self):
+        self.module = self.app.add_module(Module.new_module("Cowin Vaccination", 'en'))
+        self.app.new_form(0, "Untitled Form", None)
+        self.module.case_type = 'case'
+        suite = self.app.create_suite()
+
+        expected = """
+        <partial>
+          <remote-request>
+            <command id="cowin_search_appointment_command.m1">
+              <display>
+                <text>
+                  <locale id="cowin_search_appointment.m1"/>
+                </text>
+              </display>
+            </command>
+            <session>
+              <query url="https://cdn-api.co-vin.in/api/v2/appointment/sessions/public/findByPin" storage-instance="cowin_appointments" template="case" default_search="false">
```

Comment:
`template` should be omitted here

Reply:
https://github.com/dimagi/commcare-hq/pull/29867/commits/cf9a24f7e168868e22d2f6ad7f25ea2a175a905e